README.md

```diff
@@ -146,43 +146,41 @@ readData(dataInput) {
 ```
 
 
-# Development
+# Development and Testing
 
-## Building And Installing from Sources
+Hazelcast Node.js Client is developed using Typescript. If you want to help with bug fixes, develop new features or
```

Comment:
Let's be consistent when stating "Node.js Client". I think, in most of the document, we use it as "Node.js client" ("c" is not capitalized).